codegen/arrai/sysl.arrai

```diff
@@ -1,8 +1,20 @@
+let patterns = \node node('attrs')?('patterns')?('a')('elt').a:{} >> .('s').s => .@item;
+
 let app = (
     name: \app
         $`${app('name')('part').a >> .s::}`,
     endpoint: \app \epname
         ((app('endpoints') where .@value('name').s = epname) single).@value,
+
+    # dbConfig returns a tuple of DB config parameters specified as app attributes.
+    dbConfig: \app cond patterns(app) {{"DB", ...}: (
+        type: app('attrs')('db_type')?.@value.s:'',
+        name: app('attrs')('db_name')?.@value.s:'',
+        user: app('attrs')('db_user')?.@value.s:'',
+        password: app('attrs')('db_password')?.@value.s:'',
+        host: app('attrs')('db_host')?.@value.s:'',
+        port: app('attrs')('db_port')?.@value.s:'',
+    )},
```

Comment:
Runtime config is passed through the generated app's config file, not via Sysl definitions.